airgun/views/rhai.py

```diff
@@ -0,0 +1,138 @@
+from widgetastic_patternfly import Button, Dropdown
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import (
+    Checkbox,
+    GenericLocatorWidget,
+    ParametrizedView,
+    Table,
+    Text,
+    TextInput,
+    View
+)
+
+from airgun.views.common import BaseLoggedInView
+
+
+class Overview(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+
+    @View.nested
+    class action_summary(View):
+        title = Text(".//h2[@class='title']")
+
+    @View.nested
+    class newest_systems(View):
+        title = Text(".//h2[@class='title']")
+        table = Table(".//section[contains(normalize-space(.),"
+                      "'Newest Systems')]//table")
+
+    @View.nested
+    class optimize_your_expeirience(View):
+        title = Text(".//h2[@class='title']")
+        registered_systems = Text(".//div[@class='chart-metrics']/span")
+        stability_issues = Text(
+            ".//span[normalize-space(.)='View stability issues']"
+            "/../../div[contains(@class, 'count')]")
```

Comment:
we do not use `..` way of addressing elements. Please use `parent` or `ancestors` or any other xpath constructs